pkg/instance/service.go

```diff
@@ -74,64 +75,54 @@ type service struct {
 	kubernetesService  kubernetesExecutor
 }
 
-func (s service) LinkDeploy(token string, sourceInstance, destinationInstance *model.Instance) error {
-	err := s.link(sourceInstance, destinationInstance)
+func (s service) ConsumeParameters(source, destination *model.Instance) error {
+	sourceStack, err := s.stackService.Find(source.StackName)
 	if err != nil {
 		return err
```

Comment:
Since we are using `Find` here for source and destination stack I think we should provide this context to the error we return. Like

`fmt.Errorf("error finding stack %q of source instance: %w", source.StackName, err)`

I am using `%w` here, since we might return our not found error or similar later on. So we still want to check that the root is such an error.